mpich-LEAP_15.spec

```diff
@@ -442,8 +442,8 @@ fi
 %{p_includedir}
 %{p_libdir}/*.so
 %{p_libdir}/pkgconfig/mpich.pc
-%{p_libdir}/pkgconfig/json-c.pc
-%{p_libdir}/pkgconfig/yaksa.pc
+#%{p_libdir}/pkgconfig/json-c.pc
+#%{p_libdir}/pkgconfig/yaksa.pc
```

Comment:
well they are still installed, but im not sure why packaging was not working when those were added, so i had tried without it and it seems to have worked.
those are just 2 of other submodules, that are pulled by mpich

Reply:
Hrm.  It's odd that if they are installed, that the RPM build is passing with those commented out since the build will fail if installed-but-unpackaged files are found in the installation tree.

Reply:
it could be that they don't install the pkgconfig file anymore.

Reply:
looking at my local build, there is only mpich there:
mschaara@boro-20:~/install/mpich/lib/pkgconfig$ ls
mpich.pc

so yea they probably don't add those submodule pkg config there anymore?

Reply:
Could be I suppose.  The merge is huge so I'm not going hunting to confirm it.  :-)  I'll trust that `rpmbuild` would flag it for us if it was a problem.